find_xml_file.rb

```diff
@@ -0,0 +1,15 @@
+
+def find_xml_file(folder)
+  files = Dir.entries(folder)
+
+  xml_file = nil
+
+  files.each do |filename|
+    if filename.match "\.xml$"
+      xml_file = filename
+      break
+    end
+  end
+  
+  return xml_file
```

Comment:
I would consider replacing the `break` in `line 10` with this `return xml_file` to DRY up the code.